cms/datavis/blocks/charts.py

```diff
@@ -236,3 +236,42 @@ def get_series_data(self, value: "StructValue") -> tuple[list[list[str | int | f
             for group_name, data_points in groups.items()
         ]
         return rows, series
+
+
+class AreaChartBlock(BaseVisualisationBlock):
+    highcharts_chart_type = HighChartsChartType.AREA
+    x_axis_type = AxisType.CATEGORY
+
+    # Error codes
+    ERROR_EMPTY_CELLS = "empty_cells"
+
+    # Remove unsupported features
+    select_chart_type = None
+    show_markers = None
+    series_customisation = None
+    show_data_labels = None
+    use_stacked_layout = None
+
+    class Meta:
+        icon = "chart-area"
+
+    def clean(self, value: "StructValue") -> "StructValue":
+        value = super().clean(value)
+        value = self.clean_table_data(value)
+        return value
```

Comment:
```suggestion
        return self.clean_table_date(value)
````

Reply:
Damn right. I have updated this in 17328bb015854fedcca256580e31be125048620d.